x/stakeibc/keeper/msg_server_claim_undelegated_tokens.go

```diff
@@ -4,75 +4,75 @@ import (
 	"context"
 	"fmt"
 
+	"github.com/Stride-Labs/stride/utils"
+	recordstypes "github.com/Stride-Labs/stride/x/records/types"
+
 	"github.com/Stride-Labs/stride/x/stakeibc/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 	bankTypes "github.com/cosmos/cosmos-sdk/x/bank/types"
 )
 
-func min(a int, b int) int {
-	if a < b {
-		return a
-	}
-	return b
-}
+var min = utils.Min
```

Comment:
Looks like this is unused

Reply:
It's used below

Reply:
must be a different commit, I don't see it here https://github.com/Stride-Labs/stride/blob/6d5da1b96a45f80c2fef4c439183b8fcd23060ad/x/stakeibc/keeper/msg_server_claim_undelegated_tokens.go

Reply:
sorry, I did remove it! removing the function now